UserBundle/Document/Group.php

```diff
@@ -6,7 +6,10 @@
 use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
 
 /**
- * @ODM\Document(collection="group_document")
+ * @ODM\Document(
+ *  collection="group_document",
+ *  repositoryClass="OpenOrchestra\UserBundle\Repository\GroupRepository"
```

Comment:
Yes, it's this issue (https://github.com/open-orchestra/open-orchestra-cms-bundle/issues/774) but isn't this group document of groupBundle wich solve issue.

But this  PR https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/804 solve it